packages/blocks/src/surface-block/surface-block.ts

```diff
@@ -1,13 +1,587 @@
+import '../page-block/edgeless/edgeless-blocks-container.js';
+
+import { assertExists, Slot } from '@blocksuite/global/utils';
 import { BlockElement } from '@blocksuite/lit';
-import { nothing } from 'lit';
-import { customElement } from 'lit/decorators.js';
+import { css, html, nothing } from 'lit';
+import { customElement, property, query } from 'lit/decorators.js';
+import * as Y from 'yjs';
 
+import { EdgelessConnectorManager } from '../page-block/edgeless/connector-manager.js';
+import type { EdgelessPageBlockComponent } from '../page-block/edgeless/edgeless-page-block.js';
+import { EdgelessFrameManager } from '../page-block/edgeless/frame-manager.js';
+import { getEdgelessElement } from '../page-block/edgeless/utils/query.js';
+import { EdgelessSnapManager } from '../page-block/edgeless/utils/snap-manager.js';
+import { Batch } from './batch.js';
+import type { IBound } from './consts.js';
+import {
+  ConnectorElement,
+  ElementCtors,
+  ElementDefaultProps,
+  FrameElement,
+  type IElementCreateProps,
+  type IElementUpdateProps,
+  type IPhasorElementLocalRecord,
+  type IPhasorElementType,
+  type PhasorElement,
+  type PhasorElementType,
+} from './elements/index.js';
+import type {
+  ComputedValue,
+  HitTestOptions,
+  SurfaceElement,
+} from './elements/surface-element.js';
+import { compare } from './grid.js';
+import { Renderer } from './renderer.js';
+import { randomSeed } from './rough/math.js';
 import type { SurfaceBlockModel } from './surface-model.js';
+import { Bound, getCommonBound } from './utils/bound.js';
+import {
+  generateElementId,
+  generateKeyBetween,
+  normalizeWheelDeltaY,
+} from './utils/index.js';
+import { serializeXYWH } from './utils/xywh.js';
 
+type id = string;
```

Comment:
For better code readability, add a blank line after `type` definition.

Reply:
okay, I'll notice